src/test/java/com/epam/training/student_gregory_kubya/ProductPage.java

```diff
@@ -0,0 +1,68 @@
+package com.epam.training.student_gregory_kubya;
+
+
+import java.time.Duration;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+
+public class ProductPage {
+
+  private final WebDriverWait wait;
+
+
+  @FindBy(css = ".js-product-reference")
+  private WebElement productReferenceNumber;
+
+  @FindBy(css = ".js-product-name")
+  private WebElement productName;
+
+  @FindBy(css = "div .scope_pdpSideBar> span.js-product-price.priceItem")
+  private WebElement productPrice;
+
+  @FindBy(css = "label[for='size_M']")
+  private WebElement sizeMButton;
```

Comment:
is not easy to ready MB uppercase letters.. maybe use something like productMediumSizeButton..